test/test_typed_array.cpp

```diff
@@ -180,8 +180,9 @@ TEST_SUITE("typed_array")
             const typed_array<T> ta{array_data};
             const auto bitmap = ta.bitmap();
             REQUIRE_EQ(bitmap.size(), n - offset);
-            for (size_t i = 0; i < bitmap.size() - 1; ++i)
+            for (int32_t i = 0; i < static_cast<int32_t>(bitmap.size()) -1; ++i)
```

Comment:
What is the reason for using `int32_t`, here and bellow?

Reply:
bitmap[] argument is a range_difference_t (aka 'long long'), and so the conversion changes the signedness